### DISCUSSION TITLE
xr.Coordinates() not working in Google Colab?

### DISCUSSION THREAD

**Discussion by thedarredondo** [pydata/xarray#9259] [General] [answered]
Hi all,

When I run 
```
xr.Coordinates()
```
in Google Colab, I get the following error:

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
[<ipython-input-29-f2dc5a553dcf>](https://localhost:8080/#) in <cell line: 1>()
----> 1 xr.Coordinates()

AttributeError: module 'xarray' has no attribute 'Coordinates'

----

The source code seems fine, as far as my limited knowledge can tell. Wondering if this is some quirk of Colab.

As to why I need xr.Coordinates() to work: I'm using bambi, and xr.Coordinates() is being called when I run the following code:
```
bikes = pd.read_csv('https://raw.githubusercontent.com/thedarredondo/data-science-fundamentals/main/Data/bikes.csv')

priors ={"temperature" : bmb.Prior("Normal", mu=0,sigma=10), "sigma" : bmb.Prior("HalfNormal",sigma=10)}

model_nbb_bmb = bmb.Model("rented ~ temperature", bikes, family = "negativebinomial", priors=priors)

idata_nbb_bmb = model_nbb_bmb.fit()

bmb.interpret.plot_predictions(model_nbb_bmb, idata_nbb_bmb, "temperature", pps=True)
```

**Comment by keewis** [comment, marked as answer]
what version of `xarray` do you have installed? `Coordinates` is a somewhat recent addition, so your version of `xarray` should not be too old.

Otherwise that would be a broken install (not sure, though).

**Reply by thedarredondo** [reply]
You're right, Colab had an older version of xarray installed. Thanks for the suggestion!

Running ```!pip install git+https://github.com/pydata/xarray.git``` in Colab, as per the documentation instructions for getting the latest version of xarray, did the trick.

Here's the graph I needed, in all its glory:

![image](https://github.com/user-attachments/assets/3c092e04-a958-4ca2-a002-a171bf63f208)
